app/components/Editor.js

```diff
@@ -205,6 +200,7 @@ export default class Editor extends React.Component {
         }
       },
       imageEditable: false,
+      imageFigure: false,
```

Comment:
Without the figure, are you able to resize an image?

Reply:
Resizing is disabled but I suppose that is better than the current behaviour of losing the pasted image right? I'm not sure if it worked properly in the first place as the images would expand/contract randomly on note load. But of course, I will look into it and see if there's a way to fix it.

Reply:
Resizing is an important feature we wouldn't want to lose. So we should find a solution to this.